aptamers.py

```diff
@@ -57,6 +58,8 @@ def __init__(
         reward_norm=1.0,
         denorm_proxy=False,
     ):
+        self.max_seq_length = max_seq_length 
```

Comment:
the super init calls get_max_path_len and get_max_path_len requires self.max_seq_length and self.min_word_len. I don't get the use of self.max_path_len, it is never used ?

Reply:
I believe this was a problem in a previous version but is currently fixed, isn't it? I'll check this myself.